simcase-game/src/components/Header.js

```diff
@@ -91,18 +99,14 @@ function Header() {
     setModal(null);
   }
 
-  // const $portal = React.useMemo(() => document.getElementById("portal"), []);
-
   return (
     <header css={headerStyle}>
       <h2
```

Comment:
Where declared `headerStyle` I can't find it :thinking: If exist, You could add the style in label 'h2' like this:
```jsx
const headerStyle = {
  h2 : {
      width: "33%",
     fontSize: "1.1em"
  }
}
```

Reply:
Passed some relevant code for that part through Slack.
The idea was to have three elements in the Header component so it centers the University logo. Something I have not done is passing a title for the Header.

Reply:
Ok I understand that, but this is the other form to declare styles for don't declare in the tag.
This:
```jsx
const headerStyle = {
  ... header styles ...
  h2 : {
      width: "33%",
     fontSize: "1.1em"
  }
}
```
Is the same that this:
```jsx
<header css={headerStyle}>
  <h2 css={{
    width: "33%",
    fontSize: "1.1em"
  }}></h2>
```